### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by Wavesonics**

### CODE REVIEW

**Comment by Wavesonics** [comment]
This is ready to merge now.

I updated the README explaining what it is and how to use it.

File: .gitignore [hunk 1]
```diff
@@ -1,14 +1,2 @@
...
-# Avoid ignoring Gradle wrapper jar file (.jar files are usually ignored)
-!gradle-wrapper.jar
-
-# Cache of project
-.gradletasknamecache
-
-# # Work around https://youtrack.jetbrains.com/issue/IDEA-116898
-# gradle/wrapper/gradle-wrapper.properties
+build/*
+.gradle/*
```
Comment (chippmann) [inline .gitignore]: The .gitignore laks all the intellij specific ignores
I think we can safely assume the user to use intellij or eclipse to use so i think we should add those gitignore entries as well. I would take it from here: http://gitignore.io/

File: runBuild [hunk 2]
```diff
@@ -0,0 +1 @@
+./gradlew build
```
Comment (chippmann) [inline runBuild:6]: I would add comments on why those files are needed

File: runBuild.bat [hunk 3]
```diff
@@ -0,0 +1 @@
+call gradlew.bat build
```
Comment (chippmann) [inline runBuild.bat:7]: I would add comments on why those files are needed

**Review by chippmann — CHANGES_REQUESTED** [review]
Thanks a lot for your work!
Some minor changes needed in my opinion but otherwise great!

File: runBuild.bat (hunk 3 above)
Comment (Wavesonics) [reply runBuild.bat:7]: Solved in 5b1cfec

File: runBuild (hunk 2 above)
Comment (Wavesonics) [reply runBuild:6]: Solved in 5b1cfec

File: .gitignore (hunk 1 above)
Comment (Wavesonics) [reply .gitignore]: Solved in 52f8757

**Review by chippmann — APPROVED** [review]
Thanks a lot for the implementation!

**Pull request merged**